zenoh-kotlin/build.gradle.kts

```diff
@@ -82,24 +82,12 @@ kotlin {
             kotlinOptions.jvmTarget = "11"
         }
         testRuns["test"].executionTask.configure {
-            val zenohPaths = "../zenoh-jni/target/release"
+            val zenohPaths = "../zenoh-jni/target/debug"
```

Comment:
I think we should be building in release, any reason on why building on debug?

Reply:
Even for testing? I did it to reduce the execution time of the CI. In Rust, errors may arise in release mode that wouldn't be noticed on debug?

Reply:
My bad, I didn't notice that this was the build for testing. `debug` is fine.